Dockerfile

```diff
@@ -22,4 +25,9 @@ USER gkh
 
 RUN helm init --client-only
 
-CMD ["/bin/sh", "/data/commands.sh"]
\ No newline at end of file
+RUN git clone  https://github.com/mhyllander/helm-secrets.git -b fixes ~/helm-secrets-plugin
```

Comment:
Not relying on a fixed version? Why you chose the 'fixes' branch?

Reply:
this explains it
https://github.com/futuresimple/helm-secrets/issues/52